assets/script.js

```diff
@@ -0,0 +1,53 @@
+/* eslint no-use-before-define:["error",{"functions":false}] */
+
+let library = [];
+
+// eslint-disable-next-line no-unused-vars
+function addBook() {
+  const book = {};
+  book.author = document.getElementById('Author').value;
+  book.title = document.getElementById('Title').value;
+  library.push(book);
+  displayBooks();
+  SaveLibrary();
+}
+
+function removeBook(title) {
+  console.log(title);
+  library = library.filter((book) => book.title !== title);
```

Comment:
- [ ] [OPTIONAL] Using the title it's ok for this milestone, but remember that the convention is to use id.